core/indexer/elasticsearchDatabaseClient.go

```diff
@@ -158,8 +159,19 @@ func (dc *databaseClient) DoMultiGet(obj object, index string) (object, error) {
 		return nil, fmt.Errorf("do multi get %s", res.String())
 	}
 
+	var responseBody []byte
+	responseBody, err = ioutil.ReadAll(res.Body)
+	if err != nil {
+		log.Warn("indexer: response body",
+			"body", string(responseBody))
+		log.Warn("indexer:cannot read from response body", "error", err)
+		return nil, err
+	}
+
 	var decodedBody object
-	if err := json.NewDecoder(res.Body).Decode(&decodedBody); err != nil {
+	if err := json.Unmarshal(responseBody, &decodedBody); err != nil {
+		log.Warn("indexer: response body",
```

Comment:
Delete this warn and add it in the next Warn as additional info, like suggested above

Reply:
body